rtgui/preferences.cc

```diff
@@ -509,14 +509,14 @@ Gtk::Widget* Preferences::getProcParamsPanel ()
     Gtk::HBox* hb42 = Gtk::manage (new Gtk::HBox ());
     darkFrameDir = Gtk::manage (new Gtk::FileChooserButton (M ("PREFERENCES_DIRDARKFRAMES"), Gtk::FILE_CHOOSER_ACTION_SELECT_FOLDER));
     Gtk::Label *dfLab = Gtk::manage (new Gtk::Label (M ("PREFERENCES_DIRDARKFRAMES") + ":"));
-    hb42->pack_start (*dfLab , Gtk::PACK_SHRINK, 4 );
+    hb42->pack_start (*dfLab, Gtk::PACK_SHRINK, 4 );
     hb42->pack_start (*darkFrameDir, Gtk::PACK_EXPAND_WIDGET, 4);
     dfLabel = Gtk::manage (new Gtk::Label ("Found:"));
     Gtk::VBox* vbdf = Gtk::manage (new Gtk::VBox ());
     vbdf->pack_start ( *hb42, Gtk::PACK_SHRINK, 4);
     vbdf->pack_start ( *dfLabel, Gtk::PACK_SHRINK, 4 );
     fdf->add ( *vbdf );
-    mvbpp->pack_start ( *fdf , Gtk::PACK_SHRINK, 4);
+    mvbpp->pack_start ( *fdf, Gtk::PACK_SHRINK, 4);
```

Comment:
@Floessie do you think it would be a good idea to run all of the code files through astyle 3.0.1 to get them consistent? Then I could manually fix what needs to be fixed, like the `wbEntries` in `procparams.cc`.

Second question, would you be willing to make the changes to `rawtherapee.astylerc` based on [the docs](http://astyle.sourceforge.net/astyle.html) to improve the formatting?